pages/apim/3.x/how-tos/how-to-enable-opentracing-with-jaeger.adoc

```diff
@@ -1,102 +1,104 @@
-
-
-. MongoDB - for instructions on how to install MongoDB, click here. 
-
-. Elasticsearch - for instructions on how to install Elasticsearch, click here. 
-
-
-== How To Enable OpenTracing (Step-By-Step)
-
-=== 1. Enable OpenTracing 
-Enable OpenTracing in the `gravitee.yml` by searching for tracing, then change `enabled` from `false` to `true`. 
-image:apim/3.x/how-tos/enable-opentracing/opentracing-enable-open-tracing-value.png[Change value from false to true.]
-
-=== 2. Start The Jaeger Component 
-Start the Jaeger component with the Docker image by running this command:  
-
-`docker run -d --name jaeger \` 
-
-NOTE: This is the docker run command for installing Jaeger direct from link:/https://www.jaegertracing.io/docs/1.25/getting-started/[Jaegertracing.io] and no customizations to the configuration are included. 
-
-*PRO-TIP:* By by running this Docker command, you'll also get access to the JaegerUI. 
-
-=== 3. Confirm Jaeger Is Enabled On Docker Desktop 
-You can confirm Jaeger has been enabled and is running by checking Docker desktop. 
-
-image:apim/3.x/how-tos/enable-opentracing/jaeger-docker-desktop.png[Jaeger is running on Docker Desktop.] 
+In our example, we've used the usual Gravitee APIM dependencies (MongoDB and Elasticsearch). Need help setting up dependencies? Click the links below.
+
+* MongoDB -- to download and install MongoDB, click link:https://www.mongodb.org/downloads#production[here^].
+* Elasticsearch -- to download and install Elasticsearch, click link:https://www.elastic.co/downloads/elasticsearch[here^].
+
+== How To Enable OpenTracing
+
+This step-by-step guide explains how to configure and run OpenTracing as a Docker image. For a quickstart guide to running OpenTracing on Docker Compose, see link:/apim/3.x/apim_opentracing_in_docker.html[Enable OpenTracing^].
+
+=== 1. Enable OpenTracing
+Enable OpenTracing in the `gravitee.yml` configuration file `tracing` section, as shown in the example below.
+
+[source, shell]
+----
+tracing:
+  enabled: true
+  type: jaeger
+  jaeger:
+    host: localhost
+    port: 14250
+#    ssl:
+#     enabled: false
+----
+
+=== 2. Start The Jaeger Component
+Start the Jaeger component with the Docker image by running the Jaeger Docker command, for example:
+
+```
+docker run -d --name jaeger \
+  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
+  -p 5775:5775/udp \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 14268:14268 \
+  -p 14250:14250 \
+  -p 9411:9411 \
+  jaegertracing/all-in-one:1.27
+```
+NOTE: For the most recent documentation on starting the Jaeger Docker image, see https://www.jaegertracing.io/[Jaegertracing.io^].
+
+*PRO-TIP:* By running this Docker command, you'll also get access to the JaegerUI.
+
+=== 3. Confirm Jaeger Is Enabled On Docker Desktop
+You can confirm Jaeger has been enabled and is running in Docker Desktop.
+
+image:apim/3.x/how-tos/enable-opentracing/jaeger-docker-desktop.png[Jaeger is running on Docker Desktop].
```

Comment:
this part is more like a "tuto", the only aim is to demonstrate the feature locally.
Not sure "How to" is an appropriate section, but it should be "Try it yourself" or something like that

Reply:
moreover, for the "Confirm Jaeger is enabled on docker desktop", it was just an advice when explaining the feature during the demo.

it should not be part of the doc imho.